chapters/unitexpressions.tex

```diff
@@ -65,10 +65,24 @@ \section{The Syntax of Unit Expressions}\label{the-syntax-of-unit-expressions}
    NON-DIGIT
 \end{lstlisting}
 
-It is required that basic and derived units of the SI system are recognized, but tools are allowed to additionally support user-defined unit symbols.
+It is required that basic and derived units of the SI system are recognized and in addition some units compatible with the SI system listed below, but tools are allowed to additionally support user-defined unit symbols.
 The required unit symbols do not make use of Greek letters, but a unit such as $\Omega$ is spelled out as \lstinline!"Ohm"!.
 Similarly degree is spelled out as \lstinline!"deg"!, both on its own (for angles) and as part of \lstinline!"degC"!, \lstinline!"degF"! and \lstinline!"degRk"! for temperatures (Celsius, Fahrenheit and Rankine).
 
+It is recommended that non SI units are only used for the displayUnit attribute -- that ensures that the model is valid even if the unit is not recognized.
+
+The units compatible with the SI system that should be recognized are:
+\begin{itemize}
+\item minute "min" (1 minute=60 s)
+\item hour "h" (1 hour=3600 s)
+\item day "d" (1 day=86400 s)
+\item liter "l" and "L" (1 liter=1 dm3)
+\item electronvolt "eV" (1 electronvolt=1.602176634e-19 J)
+\item degree "deg" (1 degree=pi/180 rad)
+\item debye "debye" (1 debye = 1e-21/299792458 Cm)
```

Comment:
Formatting:
```suggestion
\item minute \lstinline!"min"! (1 minute = 60 s)
\item hour \lstinline!"h"! (1 hour = 3600 s)
\item day \lstinline!"d"! (1 day = 86400 s)
\item liter \lstinline!"l"! and \lstinline!"L"! (1 liter = 1 dm^{3})
\item electronvolt \lstinline!"eV"! (1 electronvolt = 1.602176634e-19 J)
\item degree \lstinline!"deg"! (1 degree = $\pi/180$ rad)
\item debye \lstinline!"debye"! (1 debye = 1e-21 / 299792458 Cm)
```

Reply:
I think it would be even better to use Modelica unit strings when defining relation to other units.  For example:
- 1 \lstinline!"min"! = 60 \lstinline!"s"!
- 1 \lstinline!"h"! = 60 \lstinline!"min"!

Maybe one should even express these relations in math mode…